pkg/koordlet/statesinformer/states_informer.go

```diff
@@ -226,6 +225,22 @@ func (s *statesInformer) GetAllPods() []*PodMeta {
 	return podsInformer.GetAllPods()
 }
 
+func (s *statesInformer) GetContainer(containerHashID string) *ContainerMeta {
```

Comment:
No need to declare these methods for the statesInformer.
And other modules may use the statesInformer like:

```go
si := informerGetter.GetInformer(statesinformer.StatesInformerName).(StateInformer)
```